packages/web/src/components/chat.tsx

```diff
@@ -197,7 +205,8 @@ export function ChatPanel(props: PropsType): React.JSX.Element {
 
     const response = await aiEditApp(props.app.id, query);
 
-    const plan = response.data as Array<PlanItemType>;
+    const plan = response.data as PlanType;
+    setHistory((prevHistory) => [...prevHistory, { type: 'plan', content: plan.description }]);
     if (response.data && Array.isArray(response.data)) {
```

Comment:
you changed response.data to no longer be an array, so this `if` block will never run.

I'm changing all this code, so don't worry about a fix. Just wanted to give a heads up it's probably broken right now.